src/PBot/PBot.csproj

```diff
@@ -20,7 +20,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\..\mmbot\packages\PBot.0.1.158\lib\</OutputPath>
+    <OutputPath>bin\Debug\</OutputPath>
```

Comment:
This may have been what someone wanted at some time, but it's far from a general solution :stuck_out_tongue: